.github/ISSUE_TEMPLATE/project_story_template.md

```diff
@@ -0,0 +1,19 @@
+Instructions: Replace text below with details corresponding to your story
```

Comment:
Looks like the frontmatter/metadata got deleted. Only the `label` line needed to be removed.